### ISSUE TITLE
Disable removal of core-GUI elements

### ISSUE THREAD

**Issue by jutanke** [BioroboticsLab/biotracker_gui#34]
At the moment users may remove gui elements. If this happens by accident, the user will need to remove the CFG file. We should disable the option to remove the core elements (tracker selection, video replay etc).

**Comment by hmoenck** [comment]
Yes, one of the biologists accidentally removed the gui elements and was not able to get them back. Not only she didn't know why they permanently disappeared, she didn't know how to revert this. I would consider this a bug, not an enhancement. Users do never tinker with the config files. This basically means: If you push the wrong button, the gui elements are gone forever.
